theme/themes/pastanaga/extras/blocks.less

```diff
@@ -42,6 +42,15 @@
   border-color: rgba(120, 192, 215, 0.75);
 }
 
+.block-editor-title,
```

Comment:
isn't there a generic slate editor class? They all exhibit this behavior, let's keep this simple.

Reply:
Turns out they do not! Maybe a change in the new one?